### PR TITLE
Redesigned navbar

### PR DESCRIPTION
**Pull request by RandomDev26**
Redesigned the Navbar and made it into its own component.

### CODE REVIEW

File: reagurk/src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    <li>
+                        <a href="/" target="_blank" rel="noopener noreferrer">
+                        <button type="button" class="btn signup">Join Us</button>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="/" target="_blank" rel="noopener noreferrer">
+                        <button type="button" class="btn btn-outline-light">Log In</button>
+                        </a>
+                    </li>
```
Comment (fisher60) [inline reagurk/src/components/Navbar.jsx:40]: The internal links should use the `NavLink` tag as they redirect to our React routes. The way they are programmed here treats this like a static site.

Going back to the home screen should have `target"_blank"` either, as that opens a new tab. The `a` tag should only be used when linking directly to an external site.

File: reagurk/src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export function Navbar() {
+    return (
+        <div className="content">
+            <link rel="preconnect" href="https://fonts.gstatic.com" />
+            <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet" />
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous" />
+            <div className="navbar">
+                <ul>
+                    <li>
+                        <img src={gurkultImg} className="icon" alt="The gurkan" />
```
Comment (fisher60) [inline reagurk/src/components/Navbar.jsx:16]: as a change, the gurkult logo should also link to the home page.

**Review by fisher60 — CHANGES_REQUESTED** [review]
This looks pretty good, just some minor changes>

I also think the white on yellow is *very* difficult to read, these colors should be tweaked 
image

**Comment by gustavwilliam** [comment]
We're moving on with another version of the site (and therefore NavBar). Regardless, thanks for the PR.

**Pull request closed without merging**